packages/linux-kernel/build-jool.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+
+from tomllib import loads as toml_loads
+from requests import get
+from pathlib import Path
+from subprocess import run
+
+
+# dependency modifier
+def add_depends(package_dir: str, package_name: str,
+                depends: list[str]) -> None:
+    """Add dependencies to a package
+
+    Args:
+        package_dir (str): a directory where package sources are located
+        package_name (str): a name of package
+        depends (list[str]): a list of dependencies to add
+    """
+    depends_list: str = ', '.join(depends)
+    depends_line: str = f'misc:Depends={depends_list}\n'
+
+    substvars_file = Path(f'{package_dir}/debian/{package_name}.substvars')
+    substvars_file.write_text(depends_line)
+
+
+# find kernel version and source path
+defaults_file: str = Path('../../data/defaults.toml').read_text()
+architecture_file: str = Path('../../data/architectures/amd64.toml').read_text()
+KERNEL_VER: str = toml_loads(defaults_file).get('kernel_version')
+KERNEL_FLAVOR: str = toml_loads(architecture_file).get('kernel_flavor')
+KERNEL_SRC: str = Path.cwd().as_posix() + '/linux'
+
+# define variables
+PACKAGE_NAME: str = 'jool'
+PACKAGE_VERSION: str = '4.1.9+bf4c7e3669'
+PACKAGE_DIR: str = f'{PACKAGE_NAME}-{PACKAGE_VERSION}'
+SOURCES_ARCHIVE: str = 'jool-4.1.9+bf4c7e3669.tar.gz'
+SOURCES_URL: str = f'https://github.com/NICMx/Jool/archive/bf4c7e3669672367934dc50c8b257b7790f0e27e.tar.gz'
```

Comment:
Is there a particular reason to use a commit tarball URL instead of a tag URL? Generally, a tag URL would look more obvious and easier to update.